src/connection/DbConnection.js

```diff
@@ -167,6 +168,67 @@ module.exports = (
         });
     }
 
+    async function transactionQuery(sqls, multiValues = [], label) {
+        const outputLabel = label || DEFAULT_OUTPUT_LABEL;
```

Comment:
Some defensive code here to save us grabbing a new connection, doing all the setup, checks, etc., to then realise no queries were passed and there's nothing to do.

```
if (!sqls || !sqls.length) {
    throw new Error('No queries provided');
}
```